packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -135,6 +136,52 @@ class ScrollPhysics {
     return parent.shouldAcceptUserOffset(position);
   }
 
+  /// Provides a heuristic to determine if expensive frame-bound tasks should be
+  /// deferred.
+  ///
+  /// The velocity parameter must not be null, but may be positive, negative, or
```

Comment:
update this to also mention the other parameters?

Reply:
Updated